solr/core/src/test/org/apache/solr/cloud/TestPullReplica.java

```diff
@@ -311,16 +311,24 @@ public void testAddDocs() throws Exception {
       waitForNumDocsInAllReplicas(numDocs, pullReplicas);
 
       for (Replica r : pullReplicas) {
-        try (SolrClient pullReplicaClient = getHttpSolrClient(r)) {
-          SolrQuery req = new SolrQuery("qt", "/admin/plugins", "stats", "true");
-          QueryResponse statsResponse = pullReplicaClient.query(req);
+        JettySolrRunner jetty =
+            cluster.getJettySolrRunners().stream()
+                .filter(j -> j.getBaseUrl().toString().equals(r.getBaseUrl()))
+                .findFirst()
+                .orElse(null);
```

Comment:
I suppose ideally the cluster get could the jetty by Url (a convenience method) but I dunno how often that need is